spec/controllers/tournaments_controller_spec.rb

```diff
@@ -127,40 +140,58 @@
       context 'with existing teams' do
         it 'creates a new Tournament' do
           expect do
-            post :create, params: create_data
+            post :create, params: create_playoff_tournament_data
           end.to change(Tournament, :count).by(1)
         end
 
         it 'associates the new tournament with the authenticated user' do
           expect do
-            post :create, params: create_data
+            post :create, params: create_playoff_tournament_data
           end.to change(@user.tournaments, :count).by(1)
         end
 
         it 'associates the given teams with the created tournament' do
-          post :create, params: create_data
+          post :create, params: create_playoff_tournament_data
           body = deserialize_response response
           tournament = Tournament.find(body[:id])
           expect(tournament.teams).to match_array(@teams)
         end
 
         it 'generates a playoff stage' do
-          post :create, params: create_data
+          post :create, params: create_playoff_tournament_data
           body = deserialize_response response
           tournament = Tournament.find(body[:id])
           expect(tournament.stages.first).to be_a(Stage)
         end
 
         it 'generates a playoff stage with all given teams' do
-          post :create, params: create_data
+          post :create, params: create_playoff_tournament_data
           body = deserialize_response response
           tournament = Tournament.find(body[:id])
           included_teams = tournament.stages.first.matches.map { |m| m.match_scores.map(&:team) }.flatten.uniq
           expect(included_teams).to match_array(@teams)
         end
 
+        context 'generates a group stage tournament' do
+          before(:each) do
+            post :create, params: create_group_tournament_data
+            body = deserialize_response response
+            @group_stage_tournament = Tournament.find(body[:id])
+          end
+
+          it 'with all given teams in their respective groups' do
```

Comment:
wording should be different... `it with all given teams in their respective groups` doesn't sound right

Reply:
No, but if you read the "sentence" it produces it is: 
"`TournamentsController POST #create with authentication headers with existing teams generates a group stage tournament with all given teams in their respective groups`" which sounds alright to me.

Reply:
Changed it after changing the context above